### ISSUE TITLE
Home page: Incorrect Button/navigation

### ISSUE THREAD

**Issue by Wholloran** [openforge/OPENFORGE.IO#179]
The first button on the home page says "get to know us" instead of "Get In Touch" and attempts to navigate to "/#about" rather than "/contact"

**Comment by jedihacks** [comment]
If you're referring to this one

<img width="1224" alt="screen shot 2018-07-31 at 9 31 34 am" src="https://user-images.githubusercontent.com/996175/43462607-8c2f715a-94a4-11e8-8269-3770c9b9d813.png">


Then I believe this is intentional.   The first CTA should be "Want to learn more about us?" --> "Get to Know Us" --> About 

and the second should be "Ready To Work With Us?"  --> "Get in Touch" --> Contact.

<img width="1224" alt="screen shot 2018-07-31 at 9 31 34 am" src="https://user-images.githubusercontent.com/996175/43462695-ca8a0834-94a4-11e8-8fbc-d62e6391ed68.png">

I did notice that the link doesn't work on Safari regardless, so I updated the button to link to the About page.  

Do we need to update the designs?

fix: https://github.com/openforge/OPENFORGE.IO/commit/392c95f0bdf1bac65e75558cabb51f3c418450ef
